bitcoin/src/blockdata/script/mod.rs

```diff
@@ -459,11 +460,13 @@ impl fmt::LowerHex for Script {
         fmt::LowerHex::fmt(&self.as_bytes().as_hex(), f)
     }
 }
+impl_to_hex_from_lower_hex!(Script, |script: &Script| script.0.len());
```

Comment:
Why are type annotations needed? It's not a big deal but I'm surprised they are.

Reply:
Since you'll have to modify the PR anyway, maybe you can also drop `.0` - `self` has `len` method already.